src/core_test/mpas_test_core.F

```diff
@@ -211,6 +212,22 @@ function test_core_run(domain) result(iErr)!{{{
 
       call mpas_stream_mgr_write(domain % streamManager, forceWriteNow=.true.)
 
+      !
+      ! Run mpas_test_openacc
+      !
+      #ifdef MPAS_OPENACC
```

Comment:
Also, I think it would be good to print a blank line before this test begins. That way, in case the test before this one doesn't terminate with a blank line, we still get readable log output.

Reply:
Gotcha, addressed in the most recent force push